packages/amplify-category-function/src/provider-utils/awscloudformation/service-walkthroughs/lambdaLayerWalkthrough.ts

```diff
@@ -51,31 +141,34 @@ function layerNameQuestion(context: any) {
   ];
 }
 
-function layerPermissionsQuestion() {
+function layerPermissionsQuestion(params: Permissions[]) {
   return [
     {
-      type: 'list',
+      type: 'checkbox',
       name: 'layerPermissions',
       message: 'Who should have permission to use this layer?',
       choices: [
         {
           name: 'Only the current AWS account',
           value: Permissions.private,
+          checked: _.includes(params, Permissions.private),
         },
         {
           name: 'Specific AWS accounts',
           value: Permissions.awsAccounts,
+          checked: _.includes(params, Permissions.awsAccounts),
         },
         {
           name: 'Specific AWS organization',
           value: Permissions.awsOrg,
+          checked: _.includes(params, Permissions.awsOrg),
         },
         {
           name: 'Public (everyone on AWS can use this layer)',
           value: Permissions.public,
+          checked: _.includes(params, Permissions.public),
         },
       ],
-      default: Permissions.private,
```

Comment:
I think it makes more sense if we keep the default as private and remove private as an option.